browser/ui/views/location_bar/brave_location_bar_view.cc

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/location_bar/location_bar.h"
+#include "brave/browser/ui/location_bar/brave_location_bar.h"
+
+#define LocationBar BraveLocationBar
```

Comment:
what is this for?

Reply:
We were previously overriding LocationBar, a subclass of LocationBarView via a chromium_src override in LocationBarView, but now we can do it in this class instead.